<?xml version="1.0"?>
<launch>
 <node pkg="pylon_camera" name="toru_camera" type="pylon_camera_node"> <!--  respawn="true" respawn_delay="30" -->
  <param name="camera_frame" value="/toru_camera" type="string" /> <!-- frame to which the camera is attached -->
  <param name="exposure" value="-1" type="int" />
  <param name="use_sequencer" value="false" type="bool" />
  <param name="desired_framerate" value="10.0" type="double" />
  <param name="use_brightness" value="true" type="bool" />
  <param name="magazino_cam_id"  value="TORU_0001_gripper"  type="string" />
  <param name="intrinsic_yaml_string" command="cat $(find pylon_camera)/calib/toru.yml"/>
  </node>
<!--<arg name="marker_size" default="13.0" />-->
<arg name="marker_size" default="8.0" />
<arg name="max_new_marker_error" default="0.08" />
<arg name="max_track_error" default="0.2" />
<arg name="cam_image_topic" default="/toru_camera/image_raw" />
<arg name="cam_info_topic" default="/toru_camera/camera_info" />
<arg name="output_frame" default="/toru_camera" />
<arg name="markers_topic" default="/ar_pose_marker" />
<arg name="viz_topic" default="/visualization_marker" />
<arg name="node_name" default="ar_track" />
<arg name="recalibrate_all" default="0" />
<node name="$(arg node_name)" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" >
   <remap from="ar_pose_marker" to="$(arg markers_topic)"/>
   <remap from="visualization_marker" to="$(arg viz_topic)"/>
</node>

 <node name="robot_base_transform" pkg="tf" type="static_transform_publisher" args="0.449084 -0.0107909 0.49317 0.0194728 0.929878 -0.367034 -0.0153135 /jaco_link_base /toru_camera 100" />



<!-- <node name="robot_base_transform" pkg="tf" type="static_transform_publisher" args="0.426168 0.23919 0.360211 -0.0662802 -0.89524 0.435927 -0.0641853 /toru_camera  /jaco_link_base 100" />-->
<!-- <node name="robot_base_transform" pkg="tf" type="static_transform_publisher" args="0.26 0.23919 0.360211 -0.0662802 -0.89524 0.435927 -0.0641853 /toru_camera  /jaco_link_base 100" /> -->

</launch>
